libs/service-portal/education/src/environments/environment.prod.ts

```diff
@@ -0,0 +1,6 @@
+import { getStaticEnv } from '@island.is/utils/environment'
+
```

Comment:
libs do not use the filereplacements as apps do.
So only apps have `environment.prod.ts` and `environment.ts`.

I would only use the environment.ts file and use getStaticEnv there